tests/Test/TestChannelBasedCommand.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Sylius\ShopApiPlugin\Test;
+
+use Sylius\ShopApiPlugin\Command\CommandInterface;
+
+class TestChannelBasedCommand implements CommandInterface
```

Comment:
```suggestion
final class TestChannelBasedCommand implements CommandInterface
```
It is just a test class, we don't need to provide a way to extend it